packages/web3/src/api/api-alephium.ts

```diff
@@ -354,7 +354,13 @@ export interface BuildSweepMultisig {
 }
 
 /** BuildTransaction */
-export interface BuildTransaction {
+export type BuildTransaction = DeployContract | ExecuteScript | Transfer1
```

Comment:
The name `Transfer1` and `Transfer2`, etc is due to us using `BuildTransaction.Transfer`  and `BuildTransactionResult.Transfer` at the full node side. Maybe we should create unique names for them such as `BuildTransferTransaction` and `BuildTransferTransactionResult`, etc. WDYT?